build.sh

```diff
@@ -8,6 +8,8 @@ docker build --force-rm -f go_grpc/Dockerfile -t go_grpc_test .
 
 docker build --force-rm -f cpp_grpc/Dockerfile -t cpp_grpc_test .
```

Comment:
should we call it `mt` for clarity?

Reply:
for now the convention is

`x_test` : multithread
`x_st_test`: specialized st version

but we can have either

`x_test` : generic all round rocksolid impl
OR
`x_mt_test`: specialized mt version
PLUS
`x_st_test`: specialized st version

Reply:
I think the latter seems closer to reality and more expressive - Go impl seems generic enough and scales nicely with more threads while C++ / Rust does not. Additionally just looking at the names would suggest which technologies need additional tweaking to achieve desired performance.